src/modules/cli/lib/execute-command.ts

```diff
@@ -24,13 +26,36 @@ class CliCommandEventEmitter extends EventEmitter {
 	): boolean {
 		return super.emit( event, payload );
 	}
+
+	setKillFn( fn: () => void ): void {
+		this.killFn = fn;
+	}
+
+	kill(): void {
+		this.killFn?.();
+	}
 }
 
-export function executeCliCommand( args: string[] ): CliCommandEventEmitter {
+export interface ExecuteCliCommandOptions {
+	silent?: boolean;
+}
+
+export function executeCliCommand(
+	args: string[],
+	options: ExecuteCliCommandOptions = {}
+): CliCommandEventEmitter {
```

Comment:
```suggestion
): [CliCommandEventEmitter, ChildProcess] {
```
Optional, but we could return the child process instance from this function, too. It would save us from the dance of having to set the kill function on the EventEmitter